package.json

```diff
@@ -66,6 +66,7 @@
     "fmt": "eslint . --ext .ts --config .eslintrc --fix",
     "pack": "oclif-dev pack",
     "postpack": "rm -f oclif.manifest.json",
+    "postinstall": "npm run build",
```

Comment:
🙈  Sorry to mention it, but since every keys are sorted alphabetically, maybe 'postinstall' should arrive before 'postpack' ?

Reply:
I've removed the commit. It's not necessary, it's was here for local debugging only :see_no_evil: